README.md

```diff
@@ -1,113 +1,96 @@
-> is less vulnerable to
-> [signer hijacking attacks](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792),
-> for example if a signer's token (such as cert-manager-controller) was stolen.
+Mounting and statically specifying the root CA is also an important recommended step. Without a manually specified
+root CA istio-csr defaults to trying to discover root CAs automatically, which could theoretically lead to a
+[signer hijacking attack](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792).
 
-#### Discover root CAs installation
+### Issuer or ClusterIssuer?
 
-```terminal
-$ helm repo add jetstack https://charts.jetstack.io
-$ helm repo update
-$ kubectl create namespace istio-system
-$ helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr
-```
+We'd recommend an `Issuer` unless you've got some reason to use a `ClusterIssuer`.
 
-#### Load root CAs from file ca.pem (Preferred)
+It's much easier to reason about the access controls for an `Issuer`. In addition, `Issuer`s will ultimately require less work
+to configure, since our examples in the [getting started](./docs/getting_started.md) guide focus on `Issuer`s.
 
-We use a Secret volume here to source the root CAs, however projects such as
-[secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver)
-are great options for improving security further.
+### Which Issuer Type?
 
-```terminal
-$ helm repo add jetstack https://charts.jetstack.io
-$ helm repo update
-$ kubectl create namespace istio-system
-$ kubectl create secret generic istio-root-ca --from-file=ca.pem=ca.pem -n cert-manager
-$ helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr \
-  --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
-  --set "volumeMounts[0].name=root-ca" \
-  --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
-  --set "volumes[0].name=root-ca" \
-  --set "volumes[0].secret.secretName=istio-root-ca"
-```
+Whether you choose to use an `Issuer` or a `ClusterIssuer`, you'll also need to choose the type of issuer you want such as:
 
-All helm value options can be found [here](./deploy/charts/istio-csr/README.md).
+- [CA](https://cert-manager.io/docs/configuration/ca/)
+- [Vault](https://cert-manager.io/docs/configuration/vault/)
+- or an [external issuer](https://cert-manager.io/docs/configuration/external/)
 
-### Installing Istio
+The key requirement is that arbitrary values can be placed into the `subjectAltName` (SAN) X.509 extension, since
+Istio places SPIFFE IDs there.
 
-If you are running Openshift, prepare the cluster for Istio.
-Follow instructions from Istio [platform setup
-guide](https://istio.io/latest/docs/setup/platform-setup/openshift/).
+That means that the ACME issuer **will not work** - publicly trusted certificates such as those issued by Let's Encrypt
+don't allow arbitrary entries in the SAN, for very good reasons.
```

Comment:
```suggestion
That means that the ACME issuer **will not work** — publicly trusted certificates such as those issued by Let's Encrypt
don't allow arbitrary entries in the SAN, for very good reasons.
```

Reply:
Haha, fixed just because it's you! :wink: